src/components/form/form.component.tsx

```diff
@@ -14,7 +13,36 @@ import {
   StyledFullWidthButtons,
 } from "./form.style";
 
-const Form = ({
+export interface FormProps extends SpaceProps {
+  /** Alignment of buttons */
+  buttonAlignment?: "left" | "right";
+  /** Child elements */
+  children?: React.ReactNode;
+  /** The total number of errors present in the form */
+  errorCount?: number;
+  /** Spacing between form fields, given number will be multiplied by base spacing unit (8) */
+  fieldSpacing?: 0 | 1 | 2 | 3 | 4 | 5 | 7;
```

Comment:
Looking at the type defined for `fieldSpacing` in `StyledFormProps`, I think we might have made a typo in the original `form.d.ts` file by missing out the `6`. Could you add this in please since the `StyledForm` technically supports this?